tests/test_views.py

```diff
@@ -51,7 +50,9 @@ def test_query_matching_authors(self):
         # Create ten citations to match the query and one that doesn't.
         factories.CitationFactory.create_batch(10, author='Verne, Jules')
         factories.CitationFactory(author='Dan Longshot')
```

Comment:
Were you planning to modify the authors view test_query_matching_authors test as mentioned in this comments?